src/components/DeleteBook.tsx

```diff
@@ -8,12 +10,13 @@ interface IBookProps {
 }
 const DeleteBook: FC<IBookProps> = ({ id, onBookDelete, title }) => {
   const [modal, setModal] = useState(false);
+  const [err, setErr] = useState<PostgrestError | null>();
   const Delete = async () => {
```

Comment:
Shouldn't we stick to naming functions in lower case?
```suggestion
  const delete = async () => {
```